scripts/scil_visualize_bundles_mosaic.py

```diff
@@ -0,0 +1,248 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+import argparse
+import os
+import shutil
+import nibabel as nib
+
+from scilpy.io.utils import (
+    add_overwrite_arg, assert_inputs_exist, assert_outputs_exists)
+
+from dipy.viz import actor, window
+from PIL import Image
+from PIL import ImageFont
+from PIL import ImageDraw
+
+
+def build_arg_parser():
+    parser = argparse.ArgumentParser(
+        description='Visualize bundles from a list. '
```

Comment:
Description should be formatted differently. Please see https://bitbucket.org/sciludes/scilpy/src/master/CONTRIBUTING.md (Argument parsing section)